serializers/src/glTF/2.0/babylon.glTFUtilities.ts

```diff
@@ -178,9 +178,16 @@ module BABYLON.GLTF2 {
          * Converts a Quaternion to right-handed
          * @param quaternion Source quaternion to convert to right-handed
          */
-        public static GetRightHandedQuaternionArrayFromRef(quaternion: number[]) {
+        public static _GetRightHandedQuaternionArrayFromRef(quaternion: number[]) {
              quaternion[0] *= -1;
              quaternion[1] *= -1;
         }
+
+        public static _NormalizeTangentFromRef(tangent: Vector4) {
+            const normalizedTangentComponent = new Vector3(tangent.x, tangent.y, tangent.z).normalize();

```

Comment:
nit: this will allocate a new Vector3 each time, consider doing it explicitly

Reply:
@bghgary I implemented the change to calculate this manually.